examples/three-nodes-cluster/docker/jepsen/entrypoint.sh

```
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# Start SSH agent for container lifetime
+eval $(ssh-agent -a /ssh-agent) > /dev/null
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)